builder/bundles.go

```diff
@@ -839,6 +840,79 @@ func buildFullChroot(b *Builder, set *bundleSet, packagerCmd []string, buildVers
 		}
 	}
 
+	// Resolve bundle content chroots against the full chroot. New files are copied
+	// to the full chroot and the bundle file lists are updated to contain the files
+	// within the chroot.
+	return addBundleContentChroots(set, fullDir)
+}
+
+// addBundleContentChroots resolves each bundle's content choots against the full chroot
+// and updates the bundle file lists.
+func addBundleContentChroots(set *bundleSet, fullDir string) error {
+	// Resolve bundle content chroots against the full chroot. Content chroot files
+	// that do not conflict with the full chroot are copied into the full chroot and
+	// added to the corresponding bundle's bundle-info file. Files that conflict with
+	// the full chroot will generate an error.
+	for _, bundle := range *set {
+		for chrootPath := range bundle.ContentChroots {
+			if _, err := os.Stat(chrootPath); err != nil {
+				return err
+			}
+
+			err := filepath.Walk(chrootPath, func(path string, fi os.FileInfo, err error) error {
+				if err != nil {
+					return err
+				}
+
+				filePath := strings.TrimPrefix(path, chrootPath)
```

Comment:
I think we should handle the scenario where the user provides the contents as `content(/dir/) `instead of the expected `content(/dir)`. Currently the mix will not fail but the Manifest will end up having files like `usr`, `usr/bin` instead of `/usr`, `/usr/bin`. This can cause errors in the swupd-client during installation.